reference/components/cf-monitord.markdown

```diff
@@ -133,6 +133,30 @@ Note: There is no way for force a refresh of the monitored data.
 * `cf_state.lmdb`
 * `history.lmdb`
 
+## Statistical Classes
+
+`cf-monitord` automatically defines classes based on the observation of the data
+is has collected. Classes defined are named for the measurement id (the promise
+handle in the case of custom measurement promises) with prefixes and or suffixes
+depending on the measurement.
+
+The following suffixes may be used when defining classes:
+
+* `_high` :: The last measurement was greater than the average.
+* `_low` :: The last measurement was lower than the average.
+* `_normal` ::
+* `_ldt` ::
+* `_dev1` :: The last measurement was at least 1 standard deviation higher than the average.
+* `_dev2` :: The last measurement was at least 2 standard deviations higher than the average. These classes are persistently defined for a number of minutes.
+* `_anomaly` :: The last measurement was at least 3 standard deviations than the average. These classes are persistently defined for a number of minutes.
+* `_microanomaly` :: The last measurement was at least 2 standard deviations higher than the average.
+
+The following prefixes may be used when defining classes:
+
+* `entropy_` ::
```

Comment:
@olehermanse did you have any description for this prefix?

https://github.com/cfengine/core/blob/master/cf-monitord/env_monitor.c#L934

Reply:
@nickanderson No, this is one of those things I'd be in favor of removing. Reduce the scope and ensure the things we do support are working well and documented.

Reply:
Yeah, well, when we do that we can remove it and add history for it. I assume it's something related to how much traffic is seen on a given port.